cmd/prober/write.go

```diff
@@ -163,11 +163,14 @@ func rekorWriteEndpoint(ctx context.Context, cert *x509.Certificate, priv *ecdsa
 	defer resp.Body.Close()
 	exportDataToPrometheus(resp, rekorURL, endpoint, POST, latency)
 
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("invalid status code when checking an entry in rekor: %s", resp.Status)
```

Comment:
we should probably try to read all of `resp.Body` (a `io.Reader`) and include that in this error message